tests/e2e/specs/in-game/reconnecting.spec.js

```diff
@@ -815,6 +815,7 @@ describe('Reconnecting after game is over', () => {
   });
 
   it('Dialogs persist after refreshing when game is over by conceded and opponent request rematch', () => {
+    cy.skipOnGameStateApi();
```

Comment:
Need to investigate this; skipping might actually be the wrong call at this stage.

This passed previously: https://cloud.cypress.io/projects/i8bxr8/runs/2376/test-results/499ba36f-d535-4897-9de5-0117565952ca/replay?actions=%5B%5D&browsers=%5B%5D&groups=%5B%5D&isFlaky=%5B%5D&modificationDateRange=%7B%22startDate%22%3A%221970-01-01%22%2C%22endDate%22%3A%222038-01-19%22%7D&orderBy=EXECUTION_ORDER&oses=%5B%5D&specs=%5B%5D&statuses=%5B%5D&testingTypesEnum=%5B%5D

so it is a regression in the GameState API.

Reply:
Update: This will be resolved in #1031.

Essentially, the previous non-GameState version of Concede sets the game's `lastEvent` and that's being used in the legacy version of `UserController.reLogin` which is used to get the latest game state.

We need updated logic that responds with a game's latest state even when its finished using GameStates, which is what we'll implement in #1031. For now, it's reasonable to skip these two tests